cosmoz-bottom-bar.js

```diff
@@ -107,7 +121,7 @@
 
 			_computedBarHeight: {
 				type: Number,
-				computed: '_computeComputedBarHeight(matchElementHeight, barHeight, _computedBarHeightKicker)',
```

Comment:
Yes, this is true - this is why it's called a kicker.
Changing it will cause `_computeComputedBarHeight` to run again, even though the value of the kicker itself isn't important.